keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -484,20 +483,19 @@ the container-host boundary.
 installed by the cluster admin, and AppArmor fails closed when a profile is not present.
 
 <<[UNRESOLVED]>>
+**BLOCKING**
+
 **SELinux** - (baseline)
-- (Peter Hunt) we could restrict to certain Type values i.e. ban `spc_t` (privileged) or only allow
-  `container_t` (only allowed to touch `container_file_t`, or container contents). That seems
-  consistent with how AppArmor and Seccomp work
-- (Jordan Liggitt) Remember that this admission plugin can't tell the difference between what the
-  user specified and what an earlier mutating admission plugin specified. Disallowing setting
-  selinux labels in the pod spec would also prevent another admission plugin from setting
-  constrained-but-shared-within-the-same-namespace selinux labels on the pod.
-- (Tim Allclair) It seems like we have 3 options here, none of them good:
-    1. Forbid setting any SELinux options (i.e. use the default) under the baseline (or restricted)
-       profiles. This probably breaks shared volumes.
-    2. Allow all SELinux options, and require a 3rd party controller to enforce custom SELinux
-       policy.
-    3. Add a configuration knob to allowlist (or deny) specific SELinux configurations.
+- Unset `SELinuxOptions` is allowed.
+- If options are set, they must match the following:
+  - `type` is one of:
```

Comment:
should `type` also be allowed to be unset?

it looks like openshift populates selinux with level-only by default:

https://github.com/openshift/apiserver-library-go/blob/master/pkg/securitycontextconstraints/sccmatching/matcher.go#L224-L234

Reply:
@mrunalp WDYT?

Reply:
> should type also be allowed to be unset?

`type` should be allowed to be unset and in that case the container runtime defaults to `container_t`

Reply:
Updated Pod Security Standards to match: https://github.com/kubernetes/website/pull/27653